test/kubemark/start-kubemark.sh

```diff
@@ -310,8 +310,22 @@ if [[ "${SCALEOUT_CLUSTER:-false}" == "true" ]]; then
   KUBEMARK_KUBECONFIG="${TP_KUBECONFIG}-${TENANT_PARTITION_SEQUENCE}"
 fi
 
+# master machine name format: {KUBE_GCE_ZONE}-kubemark-tp-1-master
```

Comment:
Please ask Sonya/Vinay to review kubemark script

Reply:
Sonya CRed.